tests/integration/modifiers/did-update-test.js

```diff
@@ -24,4 +25,26 @@ module('Integration | Modifier | did-update', function (hooks) {
 
     this.set('boundValue', 'update');
   });
+
+  test('it consumes tracked properties without re-invoking', async function (assert) {
+    class Context {
+      @tracked boundValue = 'initial';
+      @tracked secondaryValue = 'initial';
+    }
+
+    assert.expect(1);
+    this.context = new Context();
+
+    this.someMethod = () => {
+      // This assertion works as an assurance that we render before `secondaryValue` changes,
+      // and consumes its tag to ensure reading tracked properties won't re-trigger the modifier
+      assert.equal(this.context.secondaryValue, 'initial');
+    };
+
+    await render(hbs`<div {{did-update this.someMethod this.context.boundValue}}></div>`);
+
+    this.context.boundValue = 'update';
+    await settled();
+    this.context.secondaryValue = 'update';
```

Comment:
@mogstad for the sake of consistency and to ensure the test works as expected, do you mind to add another `await settled();` *after* assignment to `this.context.secondaryValue`?